src/pal/src/loader/module.cpp

```diff
@@ -660,6 +660,18 @@ PAL_RegisterLibraryDirect(
     return hModule;
 }
 
+// 
+VOID *
+PALAPI
+PAL_GetDirect(
```

Comment:
Can you please name it differently to be more descriptive? E.g. PAL_GetNativeHandle?

Reply:
I would include the "Library" in the name as well. `PAL_GetLibraryNativeHandle` ?